apps/wing/src/index.ts

```diff
@@ -51,6 +51,10 @@ async function main() {
       "Target platform (options: 'tf-aws', 'tf-azure', 'sim')",
       "tf-aws"
     )
+    .option(
```

Comment:
I am not entirely comfortable with exposing this as a first-class option because different targets (in the future) might have a very diverse (and distinct) set of settings, and I don't want this to become the union of all settings of all targets. I won't be a good DX.

Since this type of setting is target-specific, I feel it falls under the category of #493 (which we don't have at the moment).

Here's what I suggest as a temporary step:
1. Have the Azure target read the location from an environment variable `AZURE_LOCATION` and throw a descriptive error if it's not defined (also add some documentation in the --help of `wing compile`).
1. Add a comment to #493 describing this temporary workaround, so that we remember to address this when we introduce infra policies.

I think that eventually the DX is going to be something like a config file that you will pass to the compiler with target-specific settings (which will likely be passed into the app).

For now, an environment variable is likely a good enough solution.

Reply:
Yea reviewing back I think this makes total sense to just use an environment variable as a work around, since we just need someway to do this until we have Policy as Infrastructure.